### PR TITLE
Bootstrap project

### LINKED ISSUE #1: Bootstrap project
Bootstrap a simple React app with project boilerplate (lint, tests, CI).

### PR DESCRIPTION
**Pull request by jodator**
This PR bootstraps the React project & CI workflow. Closes #1.

### CODE REVIEW

File: packages/ui/.prettierrc [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "semi": false,
+  "singleQuote": true,
+  "printWidth": 120,
+  "bracketSpacing": true,
+  "overrides": [
+    {
+      "files": "*.sol",
```
Comment (krzysztof-jelski) [inline packages/ui/.prettierrc:8]: Not needed.

File: packages/ui/package.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    "build": "webpack --mode production --progress",
+    "lint": "yarn lint:prettier --check && yarn lint:eslint",
+    "lint:fix": "yarn lint:prettier --write && yarn lint:eslint --fix",
+    "lint:eslint": "eslint './{src,test}/**/*.{ts,tsx}'",
+    "lint:prettier": "yarn prettier './{src,test}/**/*.{ts,tsx,html}'",
+    "start": "webpack serve --mode development --progress",
+    "test": "mocha"
+  },
+  "dependencies": {
+    "@types/chai": "^4.2.14",
```
Comment (krzysztof-jelski) [inline packages/ui/package.json:15]: I guess types can go to devDependencies.

File: packages/ui/package.json [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    "html-webpack-plugin": "4",
+    "jsdom": "^16.4.0",
+    "jsdom-global": "^3.0.2",
+    "mocha": "^8.2.1",
+    "mock-local-storage": "^1.1.15",
+    "prettier": "^2.2.1",
+    "ts-loader": "^8.0.14",
+    "ts-node": "^9.1.1",
+    "typescript": "^4.1.3",
+    "webpack": "4",
```
Comment (krzysztof-jelski) [inline packages/ui/package.json:41]: Quite happy with webpack 5 in [polkadot apps](https://github.com/polkadot-js/apps), don't wanna try?

File: packages/ui/test/pages/Profile.test.tsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import { Profile } from '../../src/pages/Profile/Profile'
+
+describe('UI: Profile', () => {
+  it('Shows balance', () => {
+    render(<Profile />)
```
Comment (krzysztof-jelski) [inline packages/ui/test/pages/Profile.test.tsx:7]: I recommend react testing library, but that can come later.

**Review by krzysztof-jelski — APPROVED** [review]

File: packages/ui/package.json (hunk 3 above)
Comment (jodator) [reply packages/ui/package.json:41]: Sure, AFAIR webpack 5 should work OK.

File: packages/ui/test/pages/Profile.test.tsx (hunk 4 above)
Comment (jodator) [reply packages/ui/test/pages/Profile.test.tsx:7]: Sure - I was using the same setup as a dapp boileplate. I'll consult with you later what you're using 👍

File: packages/ui/package.json (hunk 3 above)
Comment (jodator) [reply packages/ui/package.json]: Works OK

**Pull request merged**